bug

./app/household/components/BreakdownTab.tsx:67:37
Type error: Property 'percentage' does not exist on type 'PieLabelRenderProps'.

  65 |                     cy="50%"
  66 |                     labelLine={false}
> 67 |                     label={({ name, percentage }) => `${name} ${percentage.toFixed(0)}%`}
     |                                     ^
  68 |                     outerRadius={120}
  69 |                     fill="#8884d8"
  70 |                     dataKey="value"
Next.js build worker exited with code: 1 and signal: null
Error: Process completed with exit code 1.

--- a/app/household/components/BreakdownTab.tsx
+++ b/app/household/components/BreakdownTab.tsx
@@ -64,7 +64,7 @@ export default function BreakdownTab({ transactions, selectedMonth }: BreakdownT
                     cx="50%"
                     cy="50%"
                     labelLine={false}
-                    label={({ name, percentage }) => `${name} ${percentage.toFixed(0)}%`}
+                    label={({ name, payload }) => `${name} ${payload.percentage.toFixed(0)}%`}
                     outerRadius={120}
                     fill="#8884d8"
                     dataKey="value"
@@ -130,7 +130,7 @@ export default function BreakdownTab({ transactions, selectedMonth }: BreakdownT
                     cx="50%"
                     cy="50%"
                     labelLine={false}
-                    label={({ name, percentage }) => `${name} ${percentage.toFixed(0)}%`}
+                    label={({ name, payload }) => `${name} ${payload.percentage.toFixed(0)}%`}
                     outerRadius={120}
                     fill="#8884d8"
                     dataKey="value"

